### PR TITLE
[Doc] Add how to set the alarm mailbox in faq

### LINKED ISSUE #10384: [Bug] [Doc] The faq.md file is not updated in time
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

When I browsed the faq.md file, I found that the alert.properties mentioned in it does not exist in the dolphinscheduler-alter module.

### What you expected to happen

I want to try to configure the alert mailbox.

### How to reproduce

In idea, first switch to the 3.0.0-beta-1-release branch, and then open the faq.md and configuration.md files:
image
image
image

enter the dolphinscheduler-alter module:
image

### Anything else

_No response_

### Version

3.0.0-beta-1

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liubo1990**
## Purpose of the pull request
Fixed [ #10384 ](https://github.com/apache/dolphinscheduler/issues/10384#issue-1265753728)

## Brief change log
Added the steps of how to set the alarm mailbox in faq.md

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: docs/docs/zh/faq.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ A：DolphinScheduler 由 5 个服务组成，MasterServer、WorkerServer、ApiSe
 
 ## Q：系统支持哪些邮箱？
 
-A：支持绝大多数邮箱，qq、163、126、139、outlook、aliyun 等皆支持。支持 **TLS 和 SSL** 协议，可以在 alert.properties 中选择性配置
+A：支持绝大多数邮箱，qq、163、126、139、outlook、aliyun 等皆支持。支持 **TLS 和 SSL** 协议，可以在dolphinscheduler的ui中进行配置：
```
Comment (SbloodyS) [inline docs/docs/zh/faq.md:27]: Please also modify the english docs.
```docs/docs/en/faq.md```

File: docs/docs/zh/faq.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ A：DolphinScheduler 由 5 个服务组成，MasterServer、WorkerServer、ApiSe
...
+![faq](../../img/faq_alter1.png)
```
Comment (SbloodyS) [inline docs/docs/zh/faq.md]: I think we should put all this into ```docs/docs/en/guide/alert/email.md``` like any other alert channel. And refer the link to the faq.

File: docs/docs/zh/faq.md (hunk 1 above)
Comment (liubo1990) [reply docs/docs/zh/faq.md:27]: ok

File: docs/docs/zh/faq.md (hunk 2 above)
Comment (liubo1990) [reply docs/docs/zh/faq.md]: It is a good idea

**Review by zhongjiajie — CHANGES_REQUESTED** [review]
Please use the English language and latest version of DolphinScheduler for the screenshot

**Comment by zhongjiajie** [comment]
> Please use the English language and latest version of DolphinScheduler for the screenshot

Please only use English version screenshot in both English and Chinese version document @liubo1990

**Comment by liubo1990** [comment]
> > Please use the English language and latest version of DolphinScheduler for the screenshot
> 
> Please only use English version screenshot in both English and Chinese version document @liubo1990

@zhongjiajie ，ok

File: docs/docs/zh/faq.md (hunk 2 above)
Comment (zhongjiajie) [reply docs/docs/zh/faq.md]: reslove

**Comment by zhongjiajie** [comment]
img check failed https://github.com/apache/dolphinscheduler/runs/6879865864?check_suite_focus=true, please remove the img '/img/faq_alter2.png', '/img/faq_alter1.png', '/img/faq_alter3.png'

**Comment by liubo1990** [comment]
> img check failed https://github.com/apache/dolphinscheduler/runs/6879865864?check_suite_focus=true, please remove the img '/img/faq_alter2.png', '/img/faq_alter1.png', '/img/faq_alter3.png'
The corresponding image has been removed, it should be fine now

**Comment by liubo1990** [comment]
> img check failed https://github.com/apache/dolphinscheduler/runs/6879865864?check_suite_focus=true, please remove the img '/img/faq_alter2.png', '/img/faq_alter1.png', '/img/faq_alter3.png'

The corresponding image has been removed, it should be fine now.

**Review by SbloodyS — APPROVED** [review]
LGTM. Do you have any better suggestions? @zhongjiajie

**Review by zhongjiajie — APPROVED** [review]

**Comment by zhongjiajie** [comment]
> LGTM. Do you have any better suggestions? @zhongjiajie

nop, merging

**Pull request merged**